utils/docker/Dockerfile.centos:7

```diff
@@ -36,6 +36,7 @@ RUN yum install -y \
 RUN yum copr -y enable jhli/ipmctl
 RUN yum copr -y enable jhli/safeclib
 RUN yum install -y libipmctl-devel
+RUN yum install -y python-devel
```

Comment:
This one really does need to go into the big `yum install` above to keep the Docker layering sane and consistent.  You could add it to the end of:
https://github.com/daos-stack/daos/blob/7402ff416e9cf47eec9af3d0977fecee0f668550/utils/docker/Dockerfile.centos:7#L30
before the `\` of course.